### DISCUSSION TITLE
Bug: HEIF is converted to jpeg

### DISCUSSION THREAD

**Discussion by SimplyCorbett** [photoprism/photoprism#1861] [Q&A] [answered]
Using ios 15.2 (HEIF) and photosync (photoprism upload, upload heic originals) my images will randomly become jpeg instead of HEIC. I know this might be a photosync issue but want to make sure I'm covering all my bases here.

Some will remain HEIC. Most will become JPEG.

![original-1](https://user-images.githubusercontent.com/7935624/147838487-5519d1c0-44df-4c75-8af5-96641764bccd.png)

As you can see here only one image is available which is jpeg but the image on my phone is heic.

However, if I open up another image both jpeg and heic are available.

![original-2](https://user-images.githubusercontent.com/7935624/147838505-67273c9d-ac1d-423a-94ce-acf8700959d3.png)

**Comment by SimplyCorbett** [comment]
Yes, I have premium on photosync.

![IMG_1754](https://user-images.githubusercontent.com/7935624/147838630-e3f0ee70-6fb9-4a12-8850-108373134c19.PNG)

![IMG_1755](https://user-images.githubusercontent.com/7935624/147838631-79bd3468-7a45-4bde-88e1-efc27d598260.PNG)

**Comment by graciousgrey** [comment, marked as answer]
Are only new live photos affected? 

Could you send us example files to samples@photoprism.app (ideally the ones directly from your phone) ?

**Reply by SimplyCorbett** [reply]
The issue ended up being the file names. I set it to "recording time" for the names and resynced/reimported and everything worked correctly.

It's a bug in photosync itself as I can replicate the issue on SFTP, so it's not a photoprism issue.
